doc/QUIRKS.md

```diff
@@ -32,3 +36,7 @@ During development, the QA server environment defaults to using `000000` as the
 The app signature can be found in the debug notification's title:
```

Comment:
Can we also add the link to the official docs on computing the signature? https://developers.google.com/identity/sms-retriever/verify#computing_your_apps_hash_string

Reply:
Done